sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsModelFactory.cs

```diff
@@ -87,10 +87,10 @@ public static SentimentConfidenceScores SentimentConfidenceScores(double positiv
         /// <param name="sentenceSentiments">Sets the <see cref="DocumentSentiment.Sentences"/> property.</param>
         /// <param name="warnings">Sets the <see cref="DetectedLanguage.Warnings"/> property.</param>
         /// <returns>A new instance of <see cref="TextAnalytics.DocumentSentiment"/> for mocking purposes.</returns>
-        public static DocumentSentiment DocumentSentiment(TextSentiment sentiment, double positiveScore, double neutralScore, double negativeScore, List<SentenceSentiment> sentenceSentiments, IList<TextAnalyticsWarning> warnings = default)
```

Comment:
@tg-msft is it a breaking change if we shipped the last GA with `List` or `IList`  and now we change it to `IEnumerable`?
If I used a `List` before, I can still pass it even though the method now takes `IEnumerable` but maybe there is something I could be missing?

Reply:
It would be a binary breaking change for anybody who's running against a newer version without a recompile.  You'd either want to stay with IList or add an [EditorBrowsable(Never)] overload that passes an IList into the method taking IEnumerable.

Reply:
Thanks @tg-msft 
Ok. so For now let's just leave `DocumentSentiment`, `SentenceSentiment`, `DetectedLanguage`, `CategorizedEntityCollection`, `KeyPhraseCollection`, and `LinkedEntityCollection` as they were